roles/usb_lib/tasks/main.yml

```diff
@@ -0,0 +1,28 @@
+- name: Install USB_LIB if 'usb_lib_installed' is not defined in {{ iiab_state_file }}
+  include_tasks: install.yml
+  when: usb_lib_install is undefined
+
+- name: Enable or Disable USB-LIB
+  include_tasks: enable-or-disable.yml
+
+- name: Enable/Disable/Restart Apache if primary
+  include_tasks: apache.yml
+  when: not nginx_enabled
+
+#- name: Enable/Disable/Restart NGINX if primary
+#  include_tasks: nginx.yml
+#  when: nginx_enabled | bool
+
+- name: Add 'usb-lib' variable values to {{ iiab_ini_file }}
+  ini_file:
+    path: "{{ iiab_ini_file }}"
+    section: usb-lib
```

Comment:
```suggestion
    section: usb_lib
```

Reply:
This will cause a dup section in iiab.ini so no